config/definition/default.yaml

```diff
@@ -109,6 +109,40 @@ spec:
         }
       }
 
+      if parameter.createService  {
+        service:{
+          apiVersion: "v1"
+          kind:       "Service"
+          metadata: {
+            name:      parameter.name
+            namespace: triggerService.namespace
+          }
+          spec:{
+              selector:{
+                "app.kubernetes.io/name": parameter.name
+                "trigger.oam.dev/name":   triggerService.name
+              }
+              ports:[
+                {
+                  name: "default"
+                  port: 80
+                  targetPort: 80
+                },
+                {
+                  for _,v in parameter.service.ports{
+                    name: v.name
+                    port: v.port
+                    targetPort: v.targetPort
+                  }
+                }]
+              if parameter.service.clusterIp != _|_ {
+                  clusterIp: parameter.service.clusterIp
```

Comment:
Why users need to specify the clusterIP?

Reply:
> Why users need to specify the clusterIP?
my fault, i will remove it.